otelcollector/configmaps/ama-metrics-settings-configmap.yaml

```diff
@@ -21,6 +21,12 @@ data:
     windowskubeproxy = false
     kappiebasic = true
     prometheuscollectorhealth = false
+    controlplane-apiserver = true
```

Comment:
comment about control plan targets being hardcoded to 30 seconds and not configurable!